<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureConnectionDialog</class>
 <widget class="QDialog" name="ConfigureConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Connection: </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Connection &amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>txtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtName"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupSqlServer">
     <property name="title">
      <string>MySQL Server</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPort">
        <property name="text">
         <string>3306</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblPassword">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>txtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtUsername">
        <property name="text">
         <string>root</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblHost">
        <property name="text">
         <string>Hos&amp;t:</string>
        </property>
        <property name="buddy">
         <cstring>txtHost</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtHost">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblUsername">
        <property name="text">
         <string>&amp;Username:</string>
        </property>
        <property name="buddy">
         <cstring>txtUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPort">
        <property name="text">
         <string>Port:</string>
        </property>
        <property name="buddy">
         <cstring>txtPort</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="chkCompression">
        <property name="text">
         <string>Enable compression</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkTunnel">
     <property name="text">
      <string>Connect through a SSH tunnel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupTunnel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>SSH Tunnel Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTunnelUsername">
        <property name="text">
         <string>Username:</string>
        </property>
        <property name="buddy">
         <cstring>txtTunnelUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtTunnelUsername"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTunnelPassword">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>txtTunnelPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtTunnelPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>You can leave this empty for Public-Key authentication</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblTunnelPort">
        <property name="text">
         <string>Port:</string>
        </property>
        <property name="buddy">
         <cstring>txtTunnelPort</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtTunnelPort">
        <property name="text">
         <string>22</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>checkTunnel</tabstop>
  <tabstop>txtTunnelUsername</tabstop>
  <tabstop>txtTunnelPassword</tabstop>
  <tabstop>txtTunnelPort</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
